doc/rtd/development/integration_tests.rst

```diff
@@ -31,26 +31,179 @@ integration tests. To run all integration tests, you would run:
 
 .. code-block:: bash
 
-    $ tox -e integration-tests
+    tox -e integration-tests
 
-``pytest`` arguments may also be passed. For example:
+``pytest`` arguments may also be passed.
+
+Or, specify a path to run all tests inside the given file or directory:
+
+.. code-block:: bash
+
+    tox -e integration-tests tests/integration_tests/modules/test_combined.py
+
+
+.. code-block:: bash
+
+    tox -e integration-tests tests/integration_tests/modules
+
+Or, run a specific test:
 
 .. code-block:: bash
 
-    $ tox -e integration-tests tests/integration_tests/modules/test_combined.py
+    tox -e integration-tests tests/integration_tests/modules/test_combined.py::test_bootcmd
+
+TODO: Maybe make these 3 commands in one tab group??
+
 
 Configuration
 =============
 
 All possible configuration values are defined in
-`tests/integration_tests/integration_settings.py`_. Defaults can be overridden
+`tests/integration_tests/integration_settings.py`_. Look in this file for
+the full list of variables that are available and for context on what each
+variable does and what the default values are.
 by supplying values in :file:`tests/integration_tests/user_settings.py` or by
 providing an environment variable of the same name prepended with
 ``CLOUD_INIT_``. For example, to set the ``PLATFORM`` setting:
 
 .. code-block:: bash
 
-    CLOUD_INIT_PLATFORM='ec2' pytest tests/integration_tests/
+    CLOUD_INIT_PLATFORM='ec2' tox -e integration_tests -- tests/integration_tests/
+
+
+Common integration test run configurations
```

Comment:
Nice explanations! For the examples, by the end they're starting to feel redundant.  I'm wondering if it'd be better to have a single example at the top and then another example at the bottom showing multiple options being set at once. Either way, I think it's ok as-is though, so if you disagree, no changes needed.

Reply:
I'd be happy to move this to a different file or to the bottom of the document or something. I agree that it starts feeling very repetitive but I do really see value in showing all the examples. So any thoughts on breaking this out into a separate page or something?

Reply:
Nah, if you think it's ok, then just leave it. You have a fresher perspective on this, so I trust you here more than me.